Simplenote/Classes/SPTagEntryField.m

```diff
@@ -22,15 +22,14 @@ - (instancetype)init
         self.textColor = [UIColor simplenoteTagViewTextColor];
         self.placeholdTextColor = [UIColor simplenoteTagViewPlaceholderColor];
         self.textAlignment = NSTextAlignmentNatural;
-        self.placeholder = NSLocalizedString(@"Add a tag...", nil);
+        self.placeholder = NSLocalizedString(@"Tag...", @"Placeholder text in textfield when adding a new tag to a note");
         self.returnKeyType = UIReturnKeyNext;
         self.autocorrectionType = UITextAutocorrectionTypeNo;
         self.autocapitalizationType = UITextAutocapitalizationTypeNone;
 
-        [self setPlaceholder:NSLocalizedString(@"Add a tag...", @"Placeholder test in textfield when adding a new tag to a note")];
```

Comment:
I took the time to remove this redundant double assignment, and to fix the "test" vs "text" typo 💅

For this change in particular, I took extra time to verify the version we get at runtime here (left) matches what we have currently on the App Store (right) 

<table>
<tr>
	<td>
<img alt="Simulator Screen Shot - iPod touch (7th generation) - 2021-09-22 at 12 16 08" src="https://user-images.githubusercontent.com/1218433/134273080-54122b84-96f9-4b75-aae3-0b423e82afc1.png" width=300/>
</td>
	<td>
<img alt="IMG_E59A16748543-1" src="https://user-images.githubusercontent.com/1218433/134273366-1c8dc521-c346-44b0-a511-94cd01f43971.jpeg" width=300/>

</td>
</table>

Reply:
I appreciate that you took the time to check this 👍 

But I'm wondering if we shouldn't keep "Add a tag..." for this string anyway. I mean, if this line of code was added afterwards, I feel like the original intent of this line was to change the wording from "Tag..." to "Add a tag..." at some point…

Besides, we should use `…` (ellipsis character) and not `...` (3 dots) in this text.

Reply:
> So… I'm a bit puzzled there.

So am I. I went through the same blame exercise while working on this.

I had a theory that could have been due to the fact that there were two instances of an `NSLocalizedString` using "Add a tag..." and that `genstrings` sort of didn't want to touch the original value then. I tested it by removing the duplication, but the value in the generated `.strings` file did not change. See 82f3e90c65e5f9fa90e483b4259cfe628e8b723c.

I think there's diminishing returns in looking into this at this point. But I think we should update the command in the other apps so they all use `xargs`, just in case there are other issues like this hiding from us in those codebases.

Reply:
This python script is gonna be moved in Ruby in our toolkit very soon anyway as I plan to work on our "localization toolchain improvements" project soon anyway (it's my next priority after my support rotation and after finishing my current project) and this migration will be part of it.